.github/workflows/libloading.yml

```diff
@@ -65,6 +65,26 @@ jobs:
         env:
           TARGET: ${{ matrix.rust_target}}
 
+  msys2-test:
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v2
+      - run: rustup install nightly --profile=minimal
+      - run: rustup default nightly
+      - uses: msys2/setup-msys2@v2
+        with:
+          release: false
+          install: gcc
+      - run: mkdir .cargo
+      - run: echo "[target.x86_64-pc-cygwin]" | Out-File -FilePath .cargo/config.toml -Append
+      - run: echo 'linker = "x86_64-pc-msys-gcc.exe"' | Out-File -FilePath .cargo/config.toml -Append
+      - run: echo "INPUT(libmsys-2.0.a)" | Out-File -FilePath "C:\msys64\usr\lib\libcygwin.a"
+      - run: cargo test --target x86_64-pc-cygwin -Zbuild-std
+        env:
+          TARGET: x86_64-pc-windows-msvc
```

Comment:
I genuinely have no idea why the `TARGET` is here, but it is even less clear why it specified `msvc` for a `pc-cygwin` target. Add a comment please.

Reply:
There's no pre-compiled std for cygwin target. rustc complains about that.

Reply:
Is `$TARGET` necessary for `-Zbuild-std`? I don't recall ever having to set it when I used `-Zbuild-std` in the past. But anyway, why is this *msvc*?

Reply:
Well... If `TARGET` is not set, it's x86_64-pc-windows-msvc by default, because that's the host triple. I'm just setting it explicitly. If you set it to x86_64-pc-cygwin, rustc will complain about lacking pre-compiled std.

Reply:
But you're passing `--target x86_64-pc-cygwin`, so what's in `$TARGET` shouldn't even come into play, should it? And if we're talking about the host triple, wouldn't that be set using `$HOST`, not `$TARGET`?

Reply:
I guess from another angle, what I'm saying is that I'm not going to pull code that contains "just because" incantations with no clear purpose. If the workflow works without `$TARGET` set -- remove it entirely (there's `--target` already). If it doesn't work without `$TARGET`, write a comment in the yaml file explaining why this target needs to be set despite `--target`, and why exactly it needs to be set to a different value, exemplifying whatever failures observed, etc etc. Otherwise this is extremely non-obvious and I would not rather spend cycles on trying to understand this in the future.

Reply:
OK. I removed this env var.